athena/Radius.tsx

```diff
@@ -81,67 +97,63 @@ function calculateRadius(
   start: Vector,
   radius: number,
   {
-    getCost,
     getResourceValue,
     getTransitionCost,
     isAccessible,
   }: RadiusConfiguration = MoveConfiguration,
 ): Map<Vector, RadiusItem> {
-  const { info } = unit;
-  const closed = new Array(map.size.width * map.size.height);
+  const closed: { [key: number]: 1 } = {};
```

Comment:
Why a number instead of boolean?

Reply:
> Why a number instead of boolean?

Heavily browser/system dependent but it is a _tiny_ bit [faster](https://perf.builder.io/?q=eyJpZCI6IjVkY2FuY2xncXlpIiwidGl0bGUiOiJjaGVjayBpZiBwcmltaXRpdmUgdmFsdWUgZXhpc3RzIiwiYmVmb3JlIjoiY29uc3Qgb2JqID0geyAyOiB0cnVlLCA3OiB0cnVlIH07XG5jb25zdCBvYmoyID0geyAyOiAxLCA3OiAxIH07IiwidGVzdHMiOlt7Im5hbWUiOiJ2YWx1ZSBleGlzdHMgdy8gYm9vbGVhbiIsImNvZGUiOiJpZiAob2JqWzJdKSB7IFxucmV0dXJuO1xufSIsInJ1bnMiOltdLCJvcHMiOjB9LHsibmFtZSI6InZhbHVlIGV4aXN0cyB3LyBpbnRlZ2VyIiwiY29kZSI6ImlmIChvYmoyWzJdKSB7XG5yZXR1cm47XG59IiwicnVucyI6W10sIm9wcyI6MH0seyJuYW1lIjoidmFsdWUgZG9lc24ndCBleGlzdCB3LyBib29sZWFuIiwiY29kZSI6ImlmIChvYmpbM10pIHtcbnJldHVybjtcbn0iLCJydW5zIjpbXSwib3BzIjowfSx7Im5hbWUiOiJ2YWx1ZSBkb2Vzbid0IGV4aXN0IHcvIGludGVnZXIiLCJjb2RlIjoiaWYgKG9iajJbM10pIHtcbnJldHVybjtcbn0iLCJydW5zIjpbXSwib3BzIjowfV0sInVwZGF0ZWQiOiIyMDI0LTA3LTA4VDAyOjE2OjI4LjM0N1oifQ%3D%3D) presumably because of some V8 optimization. In theory a BitArray would be the most efficient for the case, but in practice it ended up being slower